_posts/2018-05-08-CocoaPods-1.6.0-beta.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "CocoaPods 1.6.0 Beta Begins!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.6.0_ release will focus on stability, performance and scalability for large projects through a longer beta cycle.
+
+<!-- more -->
+
+We are happy to announce that the first beta of 1.6.0 is out! For this release, we are revamping the build settings generation process to be faster and easier to maintain going forward and we've enhanced integration of test specs!
+
+### Revamped Build Settings
+
+The generation of build settings (which reside within the `.xcconfig` files CocoaPods generates) has been completely re-written. The primary goal for this re-write was to clean up the code and make it scale much better for larger projects. 
+
+When contributors to CocoaPods make build setting changes it would often cause a cascading amount of performance regressions and trigger an unusual combination of edge cases. This made it very hard to test these changes and be certain that each fix works correctly for everyone.
+
+From a user point of view, this re-write should have zero effects when you compile your code. If there is anything that does not look right please file an [issue](https://github.com/CocoaPods/CocoaPods/issues/new) with us providing steps and a sample app showing the problem.
+
+#### LinkedIn Performance Case Study?
+
+To demonstrate the results of this re-write we worked closely with our friends at LinkedIn, who use CocoaPods on a large project. The re-write, along with some other performance improvements, have decreased their `pod install` times down to approximately 40 seconds compared to the original time of 3 minutes! Yes it was this bad.
+
+This change, while mostly invisible, helps us guarantee that CocoaPods can scale with your app as it grows.
+
+# TODO GET PERMISSION FROM LINKEDIN TO POST THIS.
+
+### Enhanced Test Spec Integration
+
+Test specs are becoming a fundamental piece of CocoaPods for pod authors. The ability to specify your test sources and have them execute through `lint` lets the feature become an invaluable tool for using the `podspec` file as a source of truth for your library. With 1.6.0, specifying multiple `test_spec` entries in your podspec will no longer be merged together into a single test bundle target but instead generate a unit test bundle for each one.
```

Comment:
s/specifying multiple `test_spec` entries in your podspec will no longer be merged together into/multiple  `test_spec` entries will no longer generate/